### PR TITLE
[FEATURE] Post validations to registry HTTP service

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-426
+r

### CODE REVIEW

File: lib/http/validation_tracker.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import ValidationTracker from '../validation_tracker'
+import request from 'request'
```
Comment (stevenzeiler) [inline lib/http/validation_tracker.js]: I usually use `superagent` for http clients but `request` works too.

File: build/monitor.js [hunk 2]
```diff
@@ -29,6 +29,14 @@ module.exports = function (options) {
 
   var trackers = [];
 
+  if (options.http) {
+    var HttpValidationTracker = require('./http/validation_tracker');
+    if (!process.env.HTTP_SERVICE_URL) {
+      return console.log('HTTP_SERVICE_URL required');
```
Comment (stevenzeiler) [inline build/monitor.js:35]: I get `HTTP_SERVICE_URL required` even without the `--http` option.

Comment (wilsonianb) [inline build/monitor.js:35]: Weird. What command are you running with?

Comment (stevenzeiler) [inline build/monitor.js:35]: `./index.js -h`

Comment (stevenzeiler) [inline build/monitor.js:35]: Oh I see the problem, you overrode the `-h` flag which previously was an alias for `--help`

Comment (stevenzeiler) [inline build/monitor.js:35]: I would always leave `-h` as an alias to `--help`, seems conventional.

File: index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ var packageJson = require('./package.json')
  
 program
   .version(packageJson.version)
+  .option('-h, --http', 'Log validations to http service')
```
Comment (stevenzeiler) [inline index.js]: remove the `-h` option, leaving `--http` without an alias

File: build/monitor.js (hunk 2 above)
Comment (wilsonianb) [inline build/monitor.js:35]: You don't think `--help` should be requiring environment variables? ;)

Comment (stevenzeiler) [inline build/monitor.js:35]: ha, I do not.

**Comment by stevenzeiler** [comment]
:+1: LGTM

**Pull request merged**